test/rest_api/test_compaction_manager.py

```diff
@@ -15,6 +15,50 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
 
+import sys
+
+from requests import HTTPError
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table
+
 def test_compaction_manager_stop_compaction(rest_api):
     resp = rest_api.send("POST", "compaction_manager/stop_compaction", { "type": "COMPACTION" })
+
+def test_compaction_manager_stop_keyspace_compaction(cql, rest_api, test_keyspace):
+    resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}", { "type": "RESHAPE" })
     resp.raise_for_status()
+
+    # non-existing keyspace
+    resp = rest_api.send("POST", f"compaction_manager/stop_keyspace_compaction/{test_keyspace}XXX", { "type": "RESHAPE" })
+    try:
+        resp.raise_for_status()
+    except HTTPError:
+        pass
```

Comment:
nitpick:
Instead of all this raise_for_status() thing, the more idiomatic pytest (and shorter code) would be to do something like
```python
assert resp.status_code = 400
```
or 
```python
assert !resp.ok
```
or something.

Reply:
there are named status codes that we can check although in some cases the test doesn't care about the exact status_code (currently we do) so checking for the error "class" via the exception type would be more useful.

Reply:
Done in #9755